src/restore.ts

```diff
@@ -105,18 +101,12 @@ async function configure(): Promise<void> {
   core.exportVariable('BUILDCACHE_DEBUG', getEnvVar('BUILDCACHE_DEBUG', '2'))
   core.exportVariable(
     'BUILDCACHE_LOG_FILE',
-    getEnvVar(
-      'BUILDCACHE_LOG_FILE',
-      path.join(cacheDir, '.buildcache', 'buildcache.log')
-    )
+    path.resolve(cacheDir, getEnvVar('BUILDCACHE_LOG_FILE', 'buildcache.log'))
```

Comment:
I'm concerned this may only work if the log file is inside the cache dir, but I believe you can configure it to be anywhere?

Maybe more like:

```suggestion
    path.resolve(getEnvVar('BUILDCACHE_LOG_FILE', path.join(cacheDir, 'buildcache.log')))
```

? (coded in the github review text box, may not compile or be valid in any way)

Reply:
If the second argument of `path.resolve` is absolute, it is returned as it is. If it is relative, it is returned based on the base directory (and with `foo/../bar` cleaned up to be `bar`).

Reply:
Ah, that is a little unexpected but if that's the way the path.resolve API works, then that works for me - verified by RTFM (on my part) and learning how path.resolve actually goes https://nodejs.org/api/path.html#path_path_resolve_paths :-)